inventory/group_vars/all.yml

```diff
@@ -74,6 +74,14 @@ bin_dir: /usr/local/bin
 #azure_vnet_name:
 #azure_route_table_name:
 
+## When OpenStack is used, if LBaaSv2 is available you can enable it with the following variables.
+#openstack_lbaas_enabled: True
+#openstack_lbaas_subnet_id: "the Neutron subnet id where the LBaaS VIP is, often same network as instances"
```

Comment:
Make the comment on the line above. The comment should say something like "Neutron subnet ID (not network ID) to create LBaaS VIP"